bookmarks/tests/test_bookmark_new_view.py

```diff
@@ -210,3 +210,16 @@ def test_should_hide_notes_if_there_are_no_notes(self):
         response = self.client.get(reverse("bookmarks:edit", args=[bookmark.id]))
 
         self.assertContains(response, '<details class="notes">', count=1)
+
+    def test_should_set_mark_unread_to_user_set_option(self):
```

Comment:
Since there is no existing test that verifies that this is not checked by default (=with a default profile), let's add one.

Reply:
Test added